core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentStatusManager.java

```diff
@@ -69,6 +74,8 @@
 
   private Configuration configuration;
 
+  private static final int READ_TABLE_STATUS_RETRY = 3;
```

Comment:
make the variable name as READ_TABLE_STATUS_RETRY_COUNT

Reply:
fixed